index.html

```diff
@@ -0,0 +1,292 @@
+              <ul>
+                <li>Item</li>
+                <li>Item</li>
+                <li>Item</li>
+                <li>Item</li>
+                <li>Item</li>
+                <li>Item</li>
+                <li>Item</li>
+                <li>Item</li>
+              </ul>
+            </div>
+            <div class="col">
+              <h1 class="guidestitle less">Guide Title</h1>
+              <ul>
+                <li>Item</li>
+                <li>Item</li>
+                <li>Item</li>
+                <li>Item</li>
+                <li>Item</li>
+                <li>Item</li>
+                <li>Item</li>
+                <li>Item</li>
+              </ul>
+            </div>
+            <div class="col">
+              <h1 class="guidestitle less">Guide Title</h1>
+              <ul>
+                <li>Item</li>
+                <li>Item</li>
+                <li>Item</li>
+                <li>Item</li>
+                <li>Item</li>
+                <li>Item</li>
+                <li>Item</li>
+                <li>Item</li>
+              </ul>
+            </div>
+            <div class="col">
+              <h1 class="guidestitle less">Guide Title</h1>
+              <ul>
+                <li>Item</li>
+                <li>Item</li>
+                <li>Item</li>
+                <li>Item</li>
+                <li>Item</li>
+                <li>Item</li>
+                <li>Item</li>
+                <li>Item</li>
+              </ul>
+            </div>
+          </div>
+          <div class="newsletter">
+            <div class="subscribe less">
+              <h1 class="important">Smashing NewsLetter</h1>
+              <div class="newsletter-img btn normal"></div>
+              <div class="newsletter-btn btn important"></div>
+            </div>
+          </div>
+          <div class="final normal">
+            <div class="final-links important">
+              <ul>
+                <li><a href="#">example link for section</a></li> <li><a href="#">example link for section</a></li> <li><a href="#">example link for section</a></li> <li><a href="#">example link for section</a></li> <li><a href="#">example link for section</a></li> <li><a href="#">example link for section</a></li> <li><a href="#">example link for section</a></li> <li><a href="#">example link for section</a></li> <li><a href="#">example link for section</a></li> <li><a href="#">example link for section</a></li> <li><a href="#">example link for section</a></li>
+              </ul>
+            </div>
+            <div class="final-img important">
+            </div>
+            <footer class="footer-links important">
+              <ul>
+                <li><a href="#">example link for section</a></li> <li><a href="#">example link for section</a></li> <li><a href="#">example link for section</a></li> <li><a href="#">example link for section</a></li> <li><a href="#">example link for section</a></li>
+              </ul>
+            </footer>
```

Comment:
The `<footer>` tag shouldn't be inside the `<main>` tag each one of them represents a different part of the website content.

Reply:
:O thank you!

Reply:
Thank you !

Reply:
Now the footer is out from the main section of the document. ✔